src/org.ts

```diff
@@ -73,3 +82,79 @@ export async function createOrg(
     result: { name: org.name },
   };
 }
+
+export async function updateOrg(
+  name: string,
+  orgData: OrgSchemaPatch,
+  options: ServiceOptions,
+): Promise<Result<undefined, OrgProblemType>> {
+  if (orgData.name !== undefined && name !== orgData.name) {
+    return {
+      didSucceed: false,
+      reason: OrgProblemType.INVALID_ORG_NAME,
+    };
+  }
+
+  const validationFailure = validateOrgData({ ...orgData }, options);
+
+  if (validationFailure !== undefined) {
+    return { didSucceed: false, reason: validationFailure };
+  }
+
+  const memberAccessType =
+    orgData.memberAccessType && MEMBER_ACCESS_TYPE_MAPPING[orgData.memberAccessType];
+
+  const orgModel = getModelForClass(OrgModelSchema, {
+    existingConnection: options.dbConnection,
+  });
+  try {
+    await orgModel.updateOne(
+      {
+        name,
+      },
+      { ...orgData, memberAccessType },
+    );
+  } catch (err) {
+    throw err as Error;
+  }
+
+  options.logger.info({ name: orgData.name }, 'Org updated');
+  return {
+    didSucceed: true,
+  };
+}
+
+export async function getOrg(
```

Comment:
I could move this method and the corresponding tests to a separate PR

Reply:
Let's just keep it in mind for the future, since for the purpose of this PR, as simple existance check would've sufficed. But since it's already here, let's keep it.